py-polars/docs/source/reference/utils.rst

```diff
@@ -0,0 +1,10 @@
+
+============
+Utils
+============
```

Comment:
Do the docs build for this? I expect this to fail because it is longer than the word.

```
=====
Utils
=====
```

Reply:
Yes, tried this locally:

![image](https://user-images.githubusercontent.com/11277667/184525309-27ad1b02-28a0-4e09-8225-5dd42e4f8cdc.png)

Reply:
Alright. :)